### PR TITLE
Feature/er 1014 clear fields js

### PR DESCRIPTION
**Pull request by martinyde**
Primary issue: 
https://jira.itkdev.dk/browse/ER-1014 - Javascript for clearing fields on problem change.
Minor fixes:
https://jira.itkdev.dk/browse/ER-1016 - Hide default help text on date.
https://jira.itkdev.dk/browse/ER-1019 - Change display of description text field

### CODE REVIEW

File: ereolen_support/ereolen_support.form.inc [hunk 1]
```diff
@@ -43,6 +43,10 @@ function ereolen_support_form($form, &$form_state) {
     [
       'type' => 'setting'
     ],
+    drupal_get_path('module', 'ereolen_support') . '/js/clearFields.js',
+    [
+      'type' => 'setting'
```
Comment (cableman) [inline ereolen_support/ereolen_support.form.inc]: The javascript here is not "settings" which is Drupal.settings in the frontend.

> Allowed values are 'file', 'inline', 'external' or 'setting'.

Think it should be file?

File: ereolen_support/js/clearFields.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+ * @file clearFields.js
+ *   Clear fields on form change.
+ */
+(function () {
+  Drupal.behaviors.ereolenClearFields = {
+    attach: function (context, settings) {
+      'use strict';
+
+      let problem = document.getElementById("edit-ereolen-support-problem");
+      clearFields();
```
Comment (cableman) [inline ereolen_support/js/clearFields.js:10]: Minor: Maybe switch the lines so the `let` statement:

```
clearFields();

let problem = document.getElementById("edit-ereolen-support-problem");
problem.addEventListener('change', function(){
        clearFields();
});
```

File: ereolen_support/js/clearFields.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+        clearFields();
+      });
+    }
+  };
+
+  /**
+   * Clear fields.
+   */
+  function clearFields() {
+    let textFields = document.querySelectorAll('.form-type-textfield input');
```
Comment (cableman) [inline ereolen_support/js/clearFields.js]: Minor: would link the let statement to be with the loop that use it an have an empty line before the comments to make it more readable.

**Review by cableman — CHANGES_REQUESTED** [review]

File: ereolen_support/ereolen_support.form.inc (hunk 1 above)
Comment (martinyde) [reply ereolen_support/ereolen_support.form.inc]: Yeah, thats fixed

File: ereolen_support/js/clearFields.js (hunk 2 above)
Comment (martinyde) [reply ereolen_support/js/clearFields.js:10]: Changed

File: ereolen_support/js/clearFields.js (hunk 3 above)
Comment (martinyde) [reply ereolen_support/js/clearFields.js]: Changed the order

**Review by cableman — APPROVED** [review]

**Pull request merged**